sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -323,6 +347,29 @@ export class ServiceBusSenderImpl implements ServiceBusSender {
       throw err;
     }
   }
+
+  private _createSendSpan(
```

Comment:
Note for future PR cleanup:

This function can be moved to modelsToBeSharedWithEventHubs.ts file as well

Reply:
Moved now